[CI] BasicDistributedJobsIT testFailOverBasics failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #11356 / windows-2022_checkpart4_platform-support-windows](https://gradle-enterprise.elastic.co/s/5uep53dylfht2)
- [elasticsearch-periodic-platform-support #11336 / windows-2022_checkpart4_platform-support-windows](https://gradle-enterprise.elastic.co/s/n4umfelzlnifo)

**Reproduction Line:**
```
gradlew ":x-pack:plugin:ml:internalClusterTest" --tests "org.elasticsearch.xpack.ml.integration.BasicDistributedJobsIT.testFailOverBasics" -Dtests.seed=C5D28FD0A8F9DB3B -Dtests.locale=haw-US -Dtests.timezone=America/Kentucky/Louisville -Druntime.java=25
```

**Applicable branches:**
9.2

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.ml.integration.BasicDistributedJobsIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testFailOverBasics'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: [job-fail-over-basics-job has last state change time [2025-10-30T13:00:19.840Z] that is more than [PT10S] behind current time [2025-10-30T13:00:30.181489300Z]
```

**Issue Reasons:**
- [9.2] 2 failures in test testFailOverBasics (1.2% fail rate in 172 executions)
- [9.2] 2 failures in step windows-2022_checkpart4_platform-support-windows (33.3% fail rate in 6 executions)
- [9.2] 2 failures in pipeline elasticsearch-periodic-platform-support (33.3% fail rate in 6 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -339,9 +339,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackIT
   method: testManyConcat
   issue: https://github.com/elastic/elasticsearch/issues/136728
-- class: org.elasticsearch.xpack.ml.integration.BasicDistributedJobsIT
-  method: testFailOverBasics
-  issue: https://github.com/elastic/elasticsearch/issues/136778
 - class: org.elasticsearch.xpack.search.CrossClusterAsyncSearchIT
   method: testRemoteClusterOnlyCCSWithFailuresOnAllShards
   issue: https://github.com/elastic/elasticsearch/issues/136894

--- a/x-pack/plugin/ml/src/internalClusterTest/java/org/elasticsearch/xpack/ml/integration/BasicDistributedJobsIT.java
+++ b/x-pack/plugin/ml/src/internalClusterTest/java/org/elasticsearch/xpack/ml/integration/BasicDistributedJobsIT.java
@@ -78,7 +78,7 @@ public class BasicDistributedJobsIT extends BaseMlIntegTestCase {
         OpenJobAction.Request openJobRequest = new OpenJobAction.Request(job.getId());
         client().execute(OpenJobAction.INSTANCE, openJobRequest).actionGet();
         awaitJobOpenedAndAssigned(job.getId(), null);
-        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(10, ChronoUnit.SECONDS), null);
+        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(30, ChronoUnit.SECONDS), null);
 
         setMlIndicesDelayedNodeLeftTimeoutToZero();
 
@@ -86,13 +86,13 @@ public class BasicDistributedJobsIT extends BaseMlIntegTestCase {
         internalCluster().stopRandomDataNode();
         ensureStableCluster(3);
         awaitJobOpenedAndAssigned(job.getId(), null);
-        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(10, ChronoUnit.SECONDS), null);
+        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(30, ChronoUnit.SECONDS), null);
 
         ensureGreen(); // replicas must be assigned, otherwise we could lose a whole index
         internalCluster().stopRandomDataNode();
         ensureStableCluster(2);
         awaitJobOpenedAndAssigned(job.getId(), null);
-        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(10, ChronoUnit.SECONDS), null);
+        assertRecentLastTaskStateChangeTime(MlTasks.jobTaskId(job.getId()), Duration.of(30, ChronoUnit.SECONDS), null);
     }
 
     @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/82591")

